src/experimentation-ess/common.js

```diff
@@ -451,4 +453,44 @@ export async function processAudit(auditURL, context, site, days) {
   return processExperimentRUMData(experimentData);
 }
 
+export async function postProcessor(auditUrl, auditData, context) {
+  const { dataAccess } = context;
+  log = context.log;
+  // iterate array auditData.auditResult
+  for (const experiment of auditData.auditResult) {
+    const experimentData = {
+      siteId: auditData.siteId,
+      experimentId: experiment.id,
+      name: experiment.label,
+      url: experiment.url,
+      status: experiment.status,
+      type: experiment.type,
+      variants: experiment.variants,
+      conversionEventName: experiment.conversionEventName,
+      conversionEventValue: experiment.conversionEventValue,
+    };
+    // eslint-disable-next-line no-await-in-loop
+    const existingExperiment = await dataAccess.getExperiment(
+      auditData.siteId,
+      experiment.id,
+      experiment.url,
+    );
+    if (existingExperiment) {
+      if (new Date(existingExperiment.startDate) < new Date(experiment.startDate)) {
+        experimentData.startDate = existingExperiment.startDate;
+      } else {
+        experimentData.startDate = experiment.startDate;
+      }
+      if (new Date(existingExperiment.endDate) > new Date(experiment.endDate)) {
```

Comment:
do we have this case?

Reply:
for some reason, if we don't have RUM data for a day or so and end date is missing from experiment page, then we don't want to override the existing end date, that way we'll consider all the RUM data for calculations as intended. This is hypothetical for now, not sure if we really have this in real data, but for completion sake, I've kept this.